src/ProgramingSummer/StringExtractor/StringExtractor.java

```diff
@@ -0,0 +1,37 @@
+package ProgramingSummer.StringExtractor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StringExtractor {
+
+    public List<Integer> findQuotes(String inText) {
+
+        List<Integer> indexFromText = new ArrayList<>();
+
+        for (int i = 0; i < inText.length(); i++) {
+
+            if (inText.charAt(i) == '"') {
+
+                indexFromText.add(i);
+            }
+        }
+        return indexFromText;
+    }
+
+    public List<String> extractString(String inText){
+
+        List<Integer> indexes = findQuotes(inText);
+        List<String> extractedStrings = new ArrayList<>();
```

Comment:
size of exctracted strings array will be sum of quotes / 2

Reply:
ahha